[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x5ccf60]

### Background

After upgrading my lnd version to master (be2566cf26e848f1f52caba9489d45822e9c28a9), lnd crashes reproducible with a segmentation violation.

### Your environment

* github master (be2566cf26e848f1f52caba9489d45822e9c28a9)
* `Linux odroid 4.14.180-178 #1 SMP PREEMPT Wed Sep 2 12:39:45 -03 2020 armv7l armv7l armv7l GNU/Linux`
* go version go1.16.7 linux/arm
* 2 GB of RAM
* bitcoind 0.17

### Steps to reproduce

* start lnd
* unlock wallet
* watch it booting up and connecting to peers for a minute until it crashes

### Expected behaviour

I expect it to not crash.

### Actual behaviour

LND starts up, connects to some peers and then after about one minute it crashes reproducible with a segfault:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x5ccf60]

goroutine 7938 [running]:
github.com/lightningnetwork/lnd/channeldb.(*OpenChannel).LoadFwdPkgs(0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
        /home/lnd-mainnet/gocode/src/github.com/lightningnetwork/lnd/channeldb/channel.go:2492 +0x4c
github.com/lightningnetwork/lnd.(*rpcServer).PendingChannels(0x1ca4180, 0x113f06c, 0x6f0d4b8, 0x6f0d4d0, 0x1ca4180, 0x1c24708, 0xfef578)
        /home/lnd-mainnet/gocode/src/github.com/lightningnetwork/lnd/rpcserver.go:3244 +0x7f4
github.com/lightningnetwork/lnd/lnrpc._Lightning_PendingChannels_Handler.func1(0x113f06c, 0x6f0d4b8, 0xd98df0, 0x6f0d4d0, 0x20, 0xd98df0, 0x6f0d4d0, 0x6f227c0)
        /home/lnd-mainnet/gocode/src/github.com/lightningnetwork/lnd/lnrpc/lightning_grpc.pb.go:2180 +0x68
github.com/lightningnetwork/lnd/rpcperms.(*InterceptorChain).middlewareUnaryServerInterceptor.func1(0x113f06c, 0x6f0d4b8, 0xd98df0, 0x6f0d4d0, 0x409be90, 0x409bea0, 0x21963d8, 0x1c24708, 0xa15ca3b0, 0x1c24718)
        /home/lnd-mainnet/gocode/src/github.com/lightningnetwork/lnd/rpcperms/interceptor.go:783 +0xf8
github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1.1.1(0x113f06c, 0x6f0d4b8, 0xd98df0, 0x6f0d4d0, 0x20, 0x0, 0x0, 0xaad864)
        /home/lnd-mainnet/gocode/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.3.0/chain.go:25 +0x4c
github.com/lightningnetwork/lnd/rpcperms.(*InterceptorChain).MacaroonUnaryServerInterceptor.func1(0x113f06c, 0x6f0d4b8, 0xd98df0, 0x6f0d4d0, 0x409be90, 0x409beb0, 0x5, 0x1c24708, 0x0, 0x1148a44)
        /home/lnd-mainnet/gocode/src/github.com/lightningnetwork/lnd/rpcperms/interceptor.go:647 +0x9c
github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1.1.1(0x113f06c, 0x6f0d4b8, 0xd98df0, 0x6f0d4d0, 0x0, 0x10, 0x8d1bc4, 0x0)
        /home/lnd-mainnet/gocode/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.3.0/chain.go:25 +0x4c
github.com/lightningnetwork/lnd/rpcperms.(*InterceptorChain).rpcStateUnaryServerInterceptor.func1(0x113f06c, 0x6f0d4b8, 0xd98df0, 0x6f0d4d0, 0x409be90, 0x409bec0, 0x409bee0, 0x10, 0x10, 0xd3e710)
        /home/lnd-mainnet/gocode/src/github.com/lightningnetwork/lnd/rpcperms/interceptor.go:738 +0x8c
github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1.1.1(0x113f06c, 0x6f0d4b8, 0xd98df0, 0x6f0d4d0, 0x10, 0x0, 0x1ee, 0x0)
        /home/lnd-mainnet/gocode/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.3.0/chain.go:25 +0x4c
github.com/lightningnetwork/lnd/rpcperms.errorLogUnaryServerInterceptor.func1(0x113f06c, 0x6f0d4b8, 0xd98df0, 0x6f0d4d0, 0x409be90, 0x409bed0, 0xd3e710, 0x409bee0, 0x0, 0x409be90)
        /home/lnd-mainnet/gocode/src/github.com/lightningnetwork/lnd/rpcperms/interceptor.go:563 +0x4c
github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1.1.1(0x113f06c, 0x6f0d4b8, 0xd98df0, 0x6f0d4d0, 0x6f2a8a0, 0x0, 0x18, 0x9ad7a4)
        /home/lnd-mainnet/gocode/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.3.0/chain.go:25 +0x4c
github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1(0x113f06c, 0x6f0d4b8, 0xd98df0, 0x6f0d4d0, 0x409be90, 0x409bea0, 0x929900, 0xdb8a68, 0x6f0d4b8, 0xdade60)
        /home/lnd-mainnet/gocode/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.3.0/chain.go:34 +0xbc
github.com/lightningnetwork/lnd/lnrpc._Lightning_PendingChannels_Handler(0xe4e0f0, 0x1ca4180, 0x113f06c, 0x6f0d4b8, 0x6f2a8a0, 0x2088c60, 0x113f06c, 0x6f0d4b8, 0x0, 0x0)
        /home/lnd-mainnet/gocode/src/github.com/lightningnetwork/lnd/lnrpc/lightning_grpc.pb.go:2182 +0x10c
google.golang.org/grpc.(*Server).processUnaryRPC(0x1cc6240, 0x11467dc, 0x21a4400, 0x65cc1e0, 0x2088d20, 0x18b059c, 0x0, 0x0, 0x0)
        /home/lnd-mainnet/gocode/pkg/mod/google.golang.org/grpc@v1.38.0/server.go:1286 +0x474
google.golang.org/grpc.(*Server).handleStream(0x1cc6240, 0x11467dc, 0x21a4400, 0x65cc1e0, 0x0)
        /home/lnd-mainnet/gocode/pkg/mod/google.golang.org/grpc@v1.38.0/server.go:1609 +0xa90
google.golang.org/grpc.(*Server).serveStreams.func1.2(0x2326810, 0x1cc6240, 0x11467dc, 0x21a4400, 0x65cc1e0)
        /home/lnd-mainnet/gocode/pkg/mod/google.golang.org/grpc@v1.38.0/server.go:934 +0x8c
created by google.golang.org/grpc.(*Server).serveStreams.func1
        /home/lnd-mainnet/gocode/pkg/mod/google.golang.org/grpc@v1.38.0/server.go:932 +0x1b8
```

--- a/rpcserver.go
+++ b/rpcserver.go
@@ -3233,22 +3233,22 @@ func (r *rpcServer) PendingChannels(ctx context.Context,
 				historical.ChanType,
 			)
 
+			// Get the number of forwarding packages from the
+			// historical channel.
+			fwdPkgs, err := historical.LoadFwdPkgs()
+			if err != nil {
+				rpcsLog.Errorf("unable to load forwarding "+
+					"packages for channel:%s, %v",
+					historical.ShortChannelID, err)
+				return nil, err
+			}
+			channel.NumForwardingPackages = int64(len(fwdPkgs))
+
 		// If the error is non-nil, and not due to older versions of lnd
 		// not persisting historical channels, return it.
 		default:
 			return nil, err
 		}
-
-		// Get the number of forwarding packages from the historical
-		// channel.
-		fwdPkgs, err := historical.LoadFwdPkgs()
-		if err != nil {
-			rpcsLog.Errorf("unable to load forwarding packages "+
-				"for channel:%s, %v",
-				historical.ShortChannelID, err)
-			return nil, err
-		}
-		channel.NumForwardingPackages = int64(len(fwdPkgs))
 
 		closeTXID := pendingClose.ClosingTXID.String()
 

